File: frontend/devTALK/src/DTO/HeaderDTO.ts
export class HeaderDTO {
	profilePicture: string;
	totalPendingConnections: number;

	constructor(
		profilePicture: string = "",
		totalPendingConnections: number = 0,
	) {
		this.profilePicture = profilePicture;
		this.totalPendingConnections = totalPendingConnections;
	}
}

File: backend/devtalk/src/main/java/com/amakakeru/devtalk/DTO/HeaderDTO.java
package com.amakakeru.devtalk.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class HeaderDTO {
	public String profilePicture;
	public long totalPendingConnections;
}
